.buildkite/pipeline.full.yml

```diff
@@ -60,26 +62,28 @@ steps:
           - "build/fixture-debug/*"
         upload: "maze_output/failed/**/*"
       docker-compose#v4.7.0:
-        pull: android-maze-runner
-        run: android-maze-runner
+        pull: android-maze-runner-bitbar
+        run: android-maze-runner-bitbar
+        service-ports: true
         command:
           - "features/smoke_tests"
           - "--tags=debug-safe"
           - "--app=/app/build/fixture-debug.apk"
-          - "--farm=bs"
-          - "--device=ANDROID_9_0"
-          - "--appium-version=1.20.2"
+          - "--farm=bb"
+          - "--device=ANDROID_9"
+          - "--no-tunnel"
+          - "--aws-public-ip"
           - "--fail-fast"
     env:
       TEST_FIXTURE_SYMBOL_DIR: "build/fixture-debug"
-    concurrency: 24
-    concurrency_group: 'browserstack-app'
+    concurrency: 25
+    concurrency_group: 'bitbar-app'
     concurrency_method: eager
 
   #
   # BitBar steps
   #
-  - label: ':bitbar: Android 10 NDK r21 end-to-end tests - batch 1'
+  - label: ':android: Android 9 NDK r21 end-to-end tests - batch 1'
```

Comment:
I had put `:browserstack:` and `:bitbar:` in deliberately to make it easy to see at a glance on the Buildkite UI where tests have been run.  I've got a ticket in with Marketing to get `:bitbar:` added to the list of custom Buildkite emojis.

I think that merge conflicts probably caused this to happen - note to myself to ensure that we branch from existing branches if appropriate to avoid the situation.